src/infiniteGrid.js

```diff
@@ -824,8 +824,10 @@ eg.module("infiniteGrid", ["jQuery", eg, window, document, "Outlayer"], function
 				$(e.target).off("load error");
 				checkCount <= 0 && callback && self.core && callback.apply(self);
 			};
+			var $el;
```

Comment:
@netil I think he(@sculove) worries about performance. But It does not make matter anymore. 
http://jindo.dev.naver.com/jsMatch/index.html?d=256&openResult=1

So I agree to remove `var $el`.

But I don't think there will be closure issue. I think $el will be remove from memory after this function is ended.